Microsoft.Toolkit.Uwp.UI.Animations/Extensions/AnimationExtensions.cs

```diff
@@ -23,7 +23,6 @@ namespace Microsoft.Toolkit.Uwp.UI.Animations
     /// </summary>
     public static partial class AnimationExtensions
     {
-        private static readonly EasingFunctionBase _defaultStoryboardEasingFunction = new CubicEase();
```

Comment:
You are right. I was thinking of having an EasingType (or InterpolationType depending on what you name it) that is public with a default value